pkg/ccl/backupccl/targets.go

```diff
@@ -493,6 +493,10 @@ func getRelevantDescChanges(
 				if _, ok := interestingParents[table.GetParentID()]; ok {
 					interestingIDs[table.GetID()] = struct{}{}
 				}
+			} else if tpe, isType := i.(sqlbase.TypeDescriptor); isType {
```

Comment:
can we just do a switch on the type and handle both of the table/type cases in the same case?